examples/circle_packing.ml

```diff
@@ -0,0 +1,105 @@
+open Joy.Shape
+
+(* global constants // RNG initialization *)
+let resolution = (1200, 900)
+let min_radius = 10
+let max_radius = 200
+let num_circles = 5000
+let max_attempts = 50000
+let _ = Stdlib.Random.self_init
+
+let palette =
+  [
+    (107, 108, 163);
+    (135, 188, 189);
+    (111, 153, 84);
+    (150, 155, 199);
+    (137, 171, 124);
+    (67, 68, 117);
+    (44, 45, 84);
```

Comment:
If it isn't too much work, it would be nice to add the color as a comment, so it's easier to use for future examples.

Reply:
Like add a comment above each tuple w/ the name of the color?